definitions/divorce/json/CaseEvent.json

```diff
@@ -2139,6 +2140,8 @@
     "DisplayOrder": 23,
     "PreConditionState(s)": "AwaitingDecreeAbsolute",
     "PostConditionState": "DARequested",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/da-requested-by-applicant",
+    "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
```

Comment:
this retry config should be  `RetriesTimeoutURLSubmittedEvent`

Reply:
well spotted @qzhou-hmcts 👍